src/lib.rs

```diff
@@ -643,7 +644,7 @@ fn read_stbl<T: Read>(f: &mut BMFFBox<T>, track: &mut Track) -> Result<()> {
                 log!("{:?}", stsd);
             }
             BoxType::TimeToSampleBox => {
-                let stts = try!(read_stts(&mut b));
+                let stts = try!(read_stts(&mut b, track));
```

Comment:
It feels more natural to me to set `track.fragmented = stts.samples.is_empty()` directly here rather than plumbing track down into `read_stts`. This keeps the read_* functions more uniform, each taking a box and returning just the structure representing that box. But it's also a general guideline in rust to avoid passing mutable state where possible.